src/content/zh-cn/fundamentals/getting-started/push-notifications/step-07.markdown

```diff
@@ -0,0 +1,82 @@
+---
+title: "从命令行发送GCM请求以发送推送通知"
+description: "Chrome的推送通知是使用谷歌云端推送(GCM).
+为了让GCM发送推送通知到你的Web客户端, 你可以从命令行发送GCM请求"
```

Comment:
This description definition uses two lines. I'm not sure that this expression is valid. I guess that you should concatenate the two lines as one line, if there is no special reason...